specs/swagger.yml

```diff
@@ -2913,6 +2913,22 @@ paths:
           $ref: '#/responses/unauthorizedResponse'
         '429':
           $ref: '#/responses/rateLimitedResponse'
+  /database/checkDataIntegrity:
```

Comment:
the API urls are all consistently kebab case not lower-camel case

`checkDataIntegrity` to `check-data-integrity`